### PR TITLE
Abort init sequence if running on PhantomJS

### PR DESCRIPTION
**Pull request by mspensieri**
@lemieux

### CODE REVIEW

File: src/js/smooch.jsx [hunk 1]
```diff
@@ -96,6 +96,8 @@ export class Smooch extends Observable {
             renderLink();
             this.trigger('ready');
             return Promise.resolve();
+        }else if (/PhantomJS/.test(navigator.userAgent) && !window.SmoochTest) {
```
Comment (lemieux) [inline src/js/smooch.jsx]: we could use instead `process.env.ENV !== 'test' and inject that value with webpack when we are doing tests.

Also, I was thinking about just adding the PhantomJS string to the regex for the link and that's it.

But yeah, we can just render nothing too.

(and formatting...)

**Comment by jmtremblay** [comment]
:+1:   Side note: I am curious what `if (/lebo|awle|pide|obo|rawli/i.test(navigator.userAgent))` is about. What are those user agents?

**Comment by alavers** [comment]
:+1:

**Comment by lemieux** [comment]
:+1:

**Pull request merged**